### PR TITLE
Update metadata.js and presence.js

### PR DESCRIPTION
**Pull request by KeremRR**
Update metadata.js and presence.js

image

image

image

### CODE REVIEW

File: RunoPW/dist/metadata.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
     "tr": "Müthiş İnsanlarla Dolu İlginç Bir Yer"
   },
   "url": "runo.pw",
-  "version": "1.0.0",
-  "logo": "https://i.imgur.com/utrvCGI.png",
-  "thumbnail": "https://i.imgur.com/f7C9hGS.png",
+  "version": "2.0",
+  "logo": "https://i.imgur.com/t06m0ly.png",
+  "thumbnail": "https://i.imgur.com/zMfRQH9.png",
```
Comment (Bas950) [inline RunoPW/dist/metadata.json:18]: Get a higher quality version of the logo if possible
and remove the white line at the bottom of the thumbnail

File: RunoPW/dist/metadata.json [hunk 2]
```diff
@@ -13,9 +13,9 @@
     "tr": "Müthiş İnsanlarla Dolu İlginç Bir Yer"
   },
   "url": "runo.pw",
-  "version": "1.0.0",
-  "logo": "https://i.imgur.com/utrvCGI.png",
-  "thumbnail": "https://i.imgur.com/f7C9hGS.png",
+  "version": "2.0",
```
Comment (Bas950) [inline RunoPW/dist/metadata.json:16]: versions are done like:
<MAJOR>,<MINOR>,<PATCH>

File: RunoPW/dist/presence.js [hunk 3]
```diff
@@ -6,20 +6,92 @@ var browsingStamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", () => {
   let presenceData = {
-    largeImageKey: "runo",
+    largeImageKey: "runo3",
+	smallImageKey: "aktif",
```
Comment (Bas950) [inline RunoPW/dist/presence.js:10]: Why is the small image key added, its all in really low quality

**Comment by KeremRR** [comment]
logo quality

**Comment by KeremRR** [comment]
versions are done like:
,,

more?

**Comment by KeremRR** [comment]
Would you check again

**Comment by KeremRR** [comment]
I removed the thumbnail

**Comment by KeremRR** [comment]
i loved it like this

**Comment by Bas950** [comment]
> i loved it like this

but the quality is bad, just revert to the old one then

**Comment by KeremRR** [comment]
please stay like this

**Comment by KeremRR** [comment]
fix it in the evening

**Comment by KeremRR** [comment]
for now please stay like this

**Comment by Bas950** [comment]
Okay your PR will stay open for 7 days, if you do not fix it within the time, then your PR will be closed.

**Comment by KeremRR** [comment]
okay, was it updated?

File: RunoPW/dist/metadata.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
   },
   "url": "runo.pw",
   "version": "2.0.0",
-  "logo": "https://i.imgur.com/t06m0ly.png",
+  "logo": "https://i.imgur.com/utrvCGI.png",
   "thumbnail": "https://i.imgur.com/zMfRQH9.png",
```
Comment (Bas950) [inline RunoPW/dist/metadata.json]: Logo is ok now,
but you still need to remove that ugly white line at the bottom of the thumbnail

**Comment by KeremRR** [comment]
okay can you look again

**Comment by KeremRR** [comment]
I can't wait for the update

**Pull request merged**